github/github_test.go

```diff
@@ -248,6 +248,52 @@ func TestNewRequest_emptyBody(t *testing.T) {
 	}
 }
 
+func TestNewRequest_errorForNoTrailingSlash(t *testing.T) {
+	tests := []struct {
+		rawurl  string
+		isError bool
+	}{
+		{"https://example.com/api/v3", true},
+		{"https://example.com/api/v3/", false},
```

Comment:
I'd suggest using keyed fields here to make the boolean values more readable. Just seeing `true` and `false` on their own is not as clear as:

```Go
{rawurl: "https://example.com/api/v3", wantError: true},
{rawurl: "https://example.com/api/v3/", wantError: false},
```

I'd also rename `isError` to `wantError`, using "want" for expected values is more consistent with style in Go tests.